openshift/solve_and_sync-template.yaml

```diff
@@ -118,10 +158,217 @@ objects:
                   template: "graph-sync"
                 arguments:
                   artifacts:
-                    - name: document
-                      from: "{{tasks.solverany.outputs.artifacts.document}}"
+                    - name: outputdocument
+                      from: "{{tasks.solverany.outputs.artifacts.outputdocument}}"
                   parameters:
                     - name: "THOTH_DOCUMENT_ID"
-                      value: "{{workflow.parameters.THOTH_SOLVER_NAME}}-{{workflow.parameters.THOTH_SOLVER_WORKFLOW_ID}}"
+                      value: "{{workflow.parameters.THOTH_SOLVER_NAME}}{{workflow.parameters.THOTH_SOLVER_WORKFLOW_ID}}"
                     - name: "THOTH_FORCE_SYNC"
                       value: "{{workflow.parameters.THOTH_FORCE_SYNC}}"
+
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_SOLVER_JOB_ID}"
+      annotations:
+        thoth-station.ninja/template-version: 0.2.0
+      labels:
+        app: thoth
+        component: solver
+        solver-type: solver-rhel-8-py36
+        operator: graph-sync
+        task: solver
+        mark: cleanup
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: solver
+            solver-type: solver-rhel-8-py36
```

Comment:
Can this become a parameter? That way we could maintain just one template for all the solver types.

Reply:
not really due to the `get_solver_names`(https://github.com/thoth-station/common/blob/5cea9e9f09457d64047d381031e79b54741e364c/thoth/common/openshift.py#L690) method used at the moment to check for solver names by different services (e.g. graph-refresh-job, metrics-exporter). We should think about an alternative solution.

Reply:
Right :+1: Hm... what about putting solvers into thoth's configmap? IMHO it worth to maintain just one solver workflow template.

Reply:
Yes solvers in Thoth CM would be a solution, there would be key, value for each solver present right?

Reply:
What about an entry like this (as CM do not support nested configuration):

```
solvers: |
   solver1
   solver2
```

The code in thoth-common would load the configmap and check for "solvers" key, subsequently parse solver names out of it:

```python
cm = _OPENSHIFT.get_configmap("thoth", namespace="thoth-infra-stage")
solvers = [s.strip() for s in cm["solvers"].splitlines()]
```

Reply:
Yes! https://github.com/thoth-station/common/pull/715